src/Pax/transactionApi.js

```diff
@@ -106,7 +106,7 @@ function getAmountInfo({ xAmount, xTip, xTax }, command) {
         case '03':
             return formatAmount(xAmount);
         case '02':
-            return [formatAmount(xAmount), formatAmount(xTax)].join(US);
+            return [formatAmount(xAmount)].join(US);
```

Comment:
do we need to remove tax now that we changed it to ''? 
If yes, let's remove the array/join.

Reply:
also remove default return values